config/value.go

```diff
@@ -229,41 +237,41 @@ func (cv Value) TryAsFloat() (float64, bool) {
 // AsString returns the configuration value as a string, or panics if not
 // string-able
 func (cv Value) AsString() string {
-	s, ok := cv.TryAsString()
-	if !ok {
-		panic(fmt.Sprintf("Can't convert to string: %v", cv.Value()))
+	if s, ok := cv.TryAsString(); ok {
+		return s
 	}
-	return s
+
+	panic(fmt.Sprintf("Can't convert to string: %v", cv.Value()))
 }
 
 // AsInt returns the configuration value as an int, or panics if not
 // int-able
 func (cv Value) AsInt() int {
-	s, ok := cv.TryAsInt()
-	if !ok {
-		panic(fmt.Sprintf("Can't convert to int: %T %v", cv.Value(), cv.Value()))
+	if s, ok := cv.TryAsInt(); ok {
+		return s
 	}
-	return s
+
+	panic(fmt.Sprintf("Can't convert to int: %T %v", cv.Value(), cv.Value()))
```

Comment:
This stuff all just feels like cleanup (up to this part anyway, I haven't read the rest of the PR). If we find any code towards the bottom that is contentious, I'd vote for extracting these simple changes into a separate PR and narrowing the scope of what we're reviewing to substantive changes only.

Personally, when I open up a 300+/- PR I have a subconscious feeling I'm going to have a bad time, and that affects the quality of my review (this is the same guidance we teach internally at Uber)